config/default.yml

```diff
@@ -813,6 +813,16 @@ Rails/SquishedSQLHeredocs:
   # to be preserved in order to work, thus auto-correction is not safe.
   SafeAutoCorrect: false
 
+Rails/TableNameAssignment:
+  Description: >-
+    Do not use `self.table_name =`. Use Inflections or `table_name_prefix` instead.
+  StyleGuide: 'https://rails.rubystyle.guide/#keep-ar-defaults'
+  Enabled: false
+  VersionAdded: '<<next>>'
+  Exclude:
+    - spec/**/*
```

Comment:
I suggest `**/spec/**/*` (if that pattern works), due to:
```
apps/spree/core/spec/lib/spree/core/number_generator_spec.rb
```

Reply:
Thanks! I changed the path.